content/docs/quickstart/go.md

```diff
@@ -7,9 +7,8 @@ description: This guide gets you started with gRPC in Go with a simple working e
 
 ### Prerequisites
 
-- Go version 1.6 or higher.
-
-For installation instructions, see Go's [Getting Started](https://golang.org/doc/install) guide.
+- **Go** version 1.9+. For installation instructions, see Go's [Getting
```

Comment:
Could we just require v1.11+ and use modules throughout this page?

Reply:
Yes.  1.9 is no longer officially supported (although we still run tests to make sure it works).  Using modules would require some changes to these instructions, but that would be appreciated.  Also, I'd recommend documenting to set `GO111MODULE=on` for consistency regardless of the version of Go.

Reply:
Great, I can update the text now before you review further if you'd like. Let me know what you prefer.

Reply:
Actually, what should be the minimum version of Go that we document as a prerequisite?

Reply:
Note that https://github.com/grpc/grpc-go/blob/master/README.md contains:

> **Prerequisites**
gRPC-Go requires Go 1.9 or later.

That should be updated too if 1.9 is no longer officially supported.

Reply:
We support the latest 3 versions of Go, so right now: 1.12, 1.13, and 1.14.

Thanks for pointing out the `README.md`.  I'll update it.

Reply:
> Great, I can update the text now before you review further if you'd like. Let me know what you prefer.

I would prefer for you to update this before review, since I think some of the changes will be pretty substantial.  Thank you!

Reply:
I'm committed WIP for this change.

Reply:
(FYI https://github.com/grpc/grpc-go/pull/3516)